books.js

```diff
@@ -38,6 +44,31 @@ class BookShelv {
     localStorage.setItem('books', JSON.stringify(this.books));
```

Comment:
- [ ] I am getting this error when loading your page, kindly update it
![image](https://user-images.githubusercontent.com/62705947/206448880-8d0c7df5-399b-44d0-ad92-3a8286d9a514.png)
- [ ] I cannot navigate from your navbar and also I cannot add a new book, kindly update it
https://www.loom.com/share/8d03e45be33c4bd98352bf0975a10d7f